routes.go

```diff
@@ -51,6 +52,27 @@ func (r PlugRoutes) action(c *gin.Context) {
 	c.Redirect(http.StatusFound, url.String())
 }
 
+func (r PlugRoutes) actionPXE(c *gin.Context) {
+
+	log.Info(c.GetHeader("X-PXE-Secret"))
+	log.Info(os.Getenv("csh_plug_pxe_secret"))
```

Comment:
It's standard practice for environment variables to be all uppercase
```suggestion
	log.Info(os.Getenv("CSH_PLUG_PXE_SECRET"))
```